keyboards/nasu/rules.mk

```diff
@@ -0,0 +1,31 @@
+# MCU name
+MCU = STM32F072
+
+# Bootloader selection
+BOOTLOADER = stm32-dfu
+
+# Wildcard to allow APM32 MCU
+DFU_SUFFIX_ARGS = -v FFFF -p FFFF
+
```

Comment:
Copy-Paste or actual intentions to use APM32?

Reply:
Intention to find whatever is on the market :)